docker-compose.yml

```
@@ -3,7 +3,7 @@ services:
   wallet:
     build: .
     ports:
-      - "3000:3000"
+      - "8701:8701"
     environment:
       - PORT=3000
```

I think we should set the port via env var instead of the `--port` flag in `package.json`. This allows people to override if they want, otherwise it is hard coded.

```suggestion
      - PORT=8701
```

---

For some reason I feel like this doesn't work. But true, it should be configurable.